experiments/speedrun/olmo_50m_gpu/olmo_50m_gpu.py

```diff
@@ -0,0 +1,47 @@
+"""
+Speedrun code for a 50M parameter model based on the Llama architecture. The model is trained on the Fineweb-Edu dataset
+(the default dataset for speedruns) on one A100.
+"""
+
+import logging
+
+from levanter.models.olmo import Olmo2Config
+
+from experiments.simple_train_config import SimpleTrainConfig
+from marin.execution.executor import executor_main
+from marin.resources import GpuConfig
+from marin.speedrun.speedrun import Author, SpeedrunConfig, default_speedrun
+
+logger = logging.getLogger("ray")
+
+olmo_50m = Olmo2Config(
+    seq_len=1024,
+    hidden_dim=192,
+    intermediate_dim=448,
+    num_heads=2,
+    num_kv_heads=2,
+    num_layers=4,
+)
+
+speedrun_config = SpeedrunConfig(
+    author=Author(
+        name="Will Held",
+        affiliation="Georgia Institute of Technology",
+        url="https://WilliamHeld.com",
+    ),
+    description="50M param model based on OLMo architecture.",
+    model_config=olmo_50m,
+    train_config=SimpleTrainConfig(
+        GpuConfig(gpu_count=1, accelerator_type="H200"),
```

Comment:
I guess we're changing both the architecture and accelerator...at least for now, I'd be interested in the data point where you only change 1

Reply:
Right now, just motivated by what hardware is on Low-Prio. Let me see if there are some A100s to run now as a control as well

Reply:
You can just add `llama_50m`